src/foremast/elb/create_elb.py

```diff
@@ -142,11 +143,35 @@ def add_listener_policy(self, json_data):
                 if stickiness.get(ext_port):
                     policies.append(stickiness.get(ext_port))
                 if policies:
-                    log.info("Adding policies: %s", policies)
+                    log.info("Adding listener policies: %s", policies)
                     elbclient.set_load_balancer_policies_of_listener(LoadBalancerName=self.app,
                                                                      LoadBalancerPort=ext_port,
                                                                      PolicyNames=policies)
 
+
+    def add_backend_policy(self, json_data):
+        """Attaches backend server policies to an ELB
+
+        Args:
+            json_data (json): return data from ELB upsert
+        """
+        env = boto3.session.Session(profile_name=self.env, region_name=self.region)
+        elbclient = env.client('elb')
+
+        #Attach backend server policies to created ELB
+        for job in json.loads(json_data)['job']:
+            for listener in job['listeners']:
+                policies = []
+                instance_port = listener['internalPort']
+                if listener['backendPolicies']:
```

Comment:
We're not accumulating `policies`, so combine the two `if` statements into one. If we have more logic in the future, then we can break it out again.
```python
policies = listener.get('backendPolicies')
if policies:
    log.info('Adding backend policies: %s', policies)
    elbclient.set(PolicyNames=policies...)
```